File: screen_timelinedetail/src/main/java/com/sarang/timelinedetail2/PictureViewHolder.java
package com.sarang.timelinedetail2;

import android.view.LayoutInflater;
import android.view.ViewGroup;

import androidx.annotation.NonNull;

import com.sarang.timelinedetail2.databinding.ItemTimelineDetailPictureBinding;


public class PictureViewHolder extends DataBindingViewHolder<ItemTimelineDetailPictureBinding> {

    public static PictureViewHolder create(ViewGroup parent) {
        LayoutInflater layoutInflater = LayoutInflater.from(parent.getContext());

        ItemTimelineDetailPictureBinding binding = ItemTimelineDetailPictureBinding
                .inflate(layoutInflater, parent, false);

        return new PictureViewHolder(binding);
    }

    public PictureViewHolder(@NonNull ItemTimelineDetailPictureBinding binding) {
        super(binding);
    }
}

File: library/src/main/java/com/sarang/feed_detail/holder/PictureViewHolder.kt
package com.sarang.feed_detail.holder

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.sarang.feed_detail.databinding.ItemTimelineDetailPictureBinding

class PictureViewHolder(val binding: ItemTimelineDetailPictureBinding) :
    RecyclerView.ViewHolder(binding.root) {
    companion object {
        fun create(parent: ViewGroup): PictureViewHolder {
            val layoutInflater = LayoutInflater.from(parent.context)
            val binding = ItemTimelineDetailPictureBinding
                .inflate(layoutInflater, parent, false)
            return PictureViewHolder(binding)
        }
    }
}